### DISCUSSION TITLE
Disable Tauri reload for specific folders

### DISCUSSION THREAD

**Discussion by adk7712** [tauri-apps/tauri#7557] [Q&A]
How can I make tauri ignore a certain folder while working in dev mode? Sometimes I would like to manipulate my folders without tauri detecting the changes and reloading the window.

**Comment by FabianLars** [comment]
You can completely disable the file watcher (for all folders) with the `--no-dev-watcher` flag. You can also add a `.taurignore` file _i think_ next to `tauri.conf.json` which uses a syntax similar to traditional `.gitignore` files so you can disable watching over that single folder.

**Reply by adk7712** [reply]
- the `--no-dev-watcher` flag does not seem to be working for me
- the `.taurignore` file works, but only for files within the src-tauri folder, is there any syntax to tell tauri to ignore files outside the src-tauri folder? say the frontend files in src folder.

**Reply by FabianLars** [reply]
- okay it was `--no-watch` (mixed it up with `--no-dev-server`) sorry.
- try prepending a `/` like `/public` or something. I'm not 100% about it's semantics either rn since i never use it.

**Reply by adk7712** [reply]
The `--no-watch` is not working for me. If you like, I can show a screen recording as well.

**Reply by FabianLars** [reply]
If you use npm it needs another `--` in front like `npm run tauri dev -- --no-watch` if that's not the issue a screen recording would indeed be appreciated.

**Reply by adk7712** [reply]
Here is a demo of the issue. (ignore the double capture)

![2023-08-08 16-03-16](https://github.com/tauri-apps/tauri/assets/73314216/a514f55c-249d-40f5-be88-c4940684cd37)

**Reply by FabianLars** [reply]
Ohhhh, i completely misunderstood the issue. I thought tauri is recompiling the window/the rust part. I didn't understand that it's about the window contents itself. In this case there's only the `--no-dev-server` flag that can turn this off, as long as Tauri is responsible for serving the files (aka. the devPath is a local file path).
